docs/specification/May2019Ballot.md

```diff
@@ -428,10 +428,33 @@ Context | Optionality | FHIR operation to generate context|  Description
 
 #### Example: 
 ```
-{ }
+{
+  "context": [
+    {
+      "key": "patient",
+      "resource": {
+        "resourceType": "Patient",
+        "id": "ewUbXT9RWEbSj5wPEdgRaBw3",
+        "identifier": [
+          {
+            "system": "urn:oid:1.2.840.114350",
```

Comment:
Are any of these identifiers that would have a URL system? If no, should we add the identifier types as examples here? EG: MRN? 

```
"identifier": [
   {
       "type": {
            "coding": [
                  {
                      "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
                      "value": "MR",
                      "display": "Medication Record Number"
                   },
                  "text": "MRN"
              ],
          }
           ....
     }, ...
],...
```